src/review/logic.py

```diff
@@ -43,14 +44,25 @@
 
 def get_reviewers(article, candidate_queryset, exclude_pks):
     prefetch_review_assignment = Prefetch(
-        'reviewer',
+        "reviewer",
         queryset=models.ReviewAssignment.objects.filter(
-            article__journal=article.journal
-        ).exclude(date_complete__isnull=True).order_by("-date_complete")
+            article__journal=article.journal,
+            date_accepted__isnull=False,
+            date_complete__isnull=False,
+        ).exclude(
+            decision="withdrawn",
```

Comment:
Looking at the above query and this one, it sounds like the business rules for a reviewassignment to be considered as "A user completed a review assignment for this paper using Janeway":

-   It has a `date_accepted`
- AND it has a `date_complete`
- AND the decision was not `"withdrawn"`

The `is_complete` being irrelevant.

If this is the case, can we do either of 

A. Store these filters into a variable (e.g `completed_reviews_filter`)
B. Use a custom manager that applies the filtering rules to the default queryset (e.g. `CompletedReviewsManager`)